src/common/cluster/cluster.ts

```diff
@@ -20,12 +20,12 @@ import { disposer, isDefined, isRequestError, toJS } from "../utils";
 import type { Response } from "request";
```

Comment:
"Update (mostly just) snapshots and a few other tests"

Please be more careful with commit names, this one does a lot more than it says. Almost skipped reviewing because it said "update snapshots"